developing/code-reviews/README.md

```diff
@@ -0,0 +1,18 @@
+# [Tools and Practice](../README.md) / Code Reviews
+
+## Overview
+
+Effective code reviews provide some of the best opportunities for learning, gaining valuable feedback from peers, and ensuring that quality standards are upheld throughout a project and defects are caught as early as possible.
+
+We recognize code reviews can be a vulnerable exercise and should be approached with care and deliberate intention.  We've collected some of the best practices for projects to systematize the code review process and lay out expectations for the creator of the pull request as well as the role of the reviewer.  Also linked is a guide to pairing because some of the same lessons overlap with code reviews even if it is happening informally or at a different stage.
+
+## Contents
+
+* [Guide to Great Code Reviews](./great-code-reviews.md)
+* [Best Practices for Submitting a Pull Request](./pull-request-best-practices.md)
+
+## Resources
+
+* [11 Proven Practices for Peer Review | IBM](https://www.ibm.com/developerworks/rational/library/11-proven-practices-for-peer-review/)
+* [Of Tracked Changes and Diffs | Truss Blog](https://truss.works/blog/2018/1/5/of-tracked-changes-and-diffs-moving-from-editorial-to-engineering)
+* [Pairing Guide](https://docs.google.com/document/d/1BlsUdxFjyXBKv8sP5k_dq16gpWW-dUJsSl-3oKDrCnc/edit)
```

Comment:
I agree we should put a lock icon on things that are not public since the playbook is. Also seems good to add another story to migrate the Pairing guide to the playbook at a later date.

Reply:
You're right I have seen the lock icon for private docs I'll add that.  Yep, I'm happy to break out the pairing guide into another story.